ProteGO/Common/DataModel/GetStatus/GetStatusResponse.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+struct GetStatusResponseBeaconId: Decodable {
+    let beaconId: String
+
+    let date: Date
+
+    enum CodingKeys: String, CodingKey {
+        case beaconId = "beacon_id"
+        case date
+    }
+}
+
+struct GetStatusResponse: Decodable {
```

Comment:
Pozostałe struktury z parametrami odpowiedzi mają nazwy z `..Result`(`RegisterDeviceResult`, `ConfirmRegistrationResult`). Proponuję pozostać przy `...Result` albo zmienić wszystkie na `...Request`.

Reply:
to bym proponował przejście na Response, bo to częściej uzywana nazwa w zapytaniach sieciowych

Reply:
Zmiena na `Response` będzie ok.

Reply:
👍